app/src/main/res/layout/content_list_fragment.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools">
+  <data>
+    <variable
+      name="contentCardAdapter"
+      type="org.oppia.app.player.content.ContentCardAdapter"/>
+  </data>
+  <RelativeLayout
```

Comment:
Prefer ConstraintLayout over RelativeLayout for performance & consistency.

Reply:
changed to constraint layout